data/examples/declaration/value/function/do/groups-comments-out.hs

```diff
@@ -0,0 +1,16 @@
+main = do
+  -- c1
+  f1
+  -- c2
+  f2
+
+  {- c3 -}
+  -- c4
+  f3
+
+  -- c5
```

Comment:
This is an issue in the comment placement logic where we never associate a comment with the previous line. Happens regardless of do notation, in cases like:

```haskell
foo = foo
-- some comment

bar = bar
```